modguard/show.py

```diff
@@ -51,7 +51,7 @@ def _recurs_build_string(str_repr: str, level: int, current: Dict[str, Any]) ->
                 str_repr = _recurs_build_string(str_repr, level + 1, next_dict)
         return str_repr
 
-    return _recurs_build_string(str_repr, 0, dict_repr)[16:]
+    return _recurs_build_string(str_repr, 0, dict_repr) + "\n"
```

Comment:
what is the change here?

Reply:
I was trying to trim the leading newline and color settings but it wasn't working consistently, so instead I'm just putting a newline before and after the output